src/main/java/org/folio/rest/handlers/ItemDeclaredLostEventHandler.java

```diff
@@ -24,26 +20,8 @@ public ItemDeclaredLostEventHandler(PostgresClient postgresClient) {
 
   @Override
   public Future<String> handle(ItemDeclaredLostEvent event) {
```

Comment:
this method is identical in 5 of 6 handlers. should it be extracted in a common parent and be overriden in `FeeFineBalanceChangedEventHandler`?

Reply:
Again, not really identical.. `event` parameters have different types, and `save()` methods are different too - there are 6 of them (there's another comment about them).